src/tracks/gosling-track/gosling-track.ts

```diff
@@ -1305,9 +1370,9 @@ const factory: PluginTrackFactory<Tile, GoslingTrackOptions> = (HGC, context, op
         hideMousePosition?: () => void; // set in HorizontalTiled1DPixiTrack
 
         /**
-         * Overrides method in HorizontalLine1DPixiTrack
+         * Called by showHoverMenu() in HiGlassComponent
          */
-        override getMouseOverHtml(mouseX: number, mouseY: number) {
+        getMouseOverHtml(mouseX: number, mouseY: number) {
```

Comment:
Here is where it is called in HiGlass 
https://github.com/higlass/higlass/blob/335f4eea6445d07baa5eda2e66077dca461b7777/app/scripts/HiGlassComponent.jsx#L4536